### PR TITLE
K8s user guide

### PR DESCRIPTION
**Pull request by Le-Zheng**

### CODE REVIEW

**Comment by Le-Zheng** [comment]
doc testing link: https://le-test-k8s.readthedocs.io/en/latest/doc/UserGuide/k8s.html

**Comment by jason-dai** [comment]
We can organize the document as follows:
1) Pull hyper-zoo image
2) Launch client docker
3) Run Analytics Zoo program
    - Client mode (init_orca_context with "k8s")
    - Cluster mode (init_orca_context with "spark-submit")
    - Scala program
4) Access logs and clear pod

File: readthedocs/source/doc/UserGuide/k8s.md [hunk 1]
```diff
@@ -32,24 +32,216 @@ sudo systemctl daemon-reload
...
+- jdk is the jdk home.
+- spark is the spark home.
+- redis is the redis home.
 
 ### **3. Run Analytics Zoo Examples on k8s**
 
 _**Note**: Please make sure `kubectl` has appropriate permission to create, list and delete pod._
 
-#### **3.1 Use `init_orca_context`**
+#### **3.1 K8s client mode (use `init_orca_context` with "k8s")**
```
Comment (jason-dai) [inline readthedocs/source/doc/UserGuide/k8s.md:112]: just "K8s Client Mode"

File: readthedocs/source/doc/UserGuide/k8s.md [hunk 2]
```diff
@@ -32,24 +32,216 @@ sudo systemctl daemon-reload
...
+init_orca_context(cluster_mode="k8s", master="k8s://https://<k8s-apiserver-host>:<k8s-apiserver-port>",
+                  container_image="intelanalytics/hyper-zoo:latest",
+                  num_nodes=2, cores=2,
+                  conf={"spark.driver.host": "x.x.x.x",
+                  "spark.driver.port": "x"})
+```
+
+Execute `python script.py` to run your program on k8s cluster directly.
+
+#### **3.2 K8s cluster mode (use `init_orca_context` with "spark-submit")**
```
Comment (jason-dai) [inline readthedocs/source/doc/UserGuide/k8s.md:128]: Just "K8s Cluster Mode"

File: readthedocs/source/doc/UserGuide/k8s.md [hunk 3]
```diff
@@ -32,24 +32,216 @@ sudo systemctl daemon-reload
...
+- --deploy-mode: submit application in client/cluster mode.
+- --name: the Spark application name.
+- --conf: to specify k8s service account, container image to use for the Spark application, driver volumes name and path, label of pods, spark driver and executor configuration, etc. You can refer to [spark configuration](https://spark.apache.org/docs/latest/configuration.html) and [spark on k8s configuration](https://spark.apache.org/docs/latest/running-on-kubernetes.html#configuration) for more details.
+- --properties-file: the customized conf properties.
+- --py-files: the extra python packages is needed.
+- --class: scala example class name.
+- --input_dir: input data path of the anomaly detection example. The data path is the mounted filesystem of the host. Refer to more details by [Kubernetes Volumes](https://spark.apache.org/docs/latest/running-on-kubernetes.html#using-kubernetes-volumes).
 
-**Run Jupyter Notebooks**
+#### **3.5 Access logs and clear pods**
```
Comment (jason-dai) [inline readthedocs/source/doc/UserGuide/k8s.md:221]: 4. Access Logs and Clear Pods*

**Pull request merged**